tests/engine/files/rule_base_segments.json

```diff
@@ -0,0 +1,62 @@
+{"ff": {"d": [], "t": -1, "s": -1},
+"rbs": {"t": -1, "s": -1, "d":
```

Comment:
I think this doesn't change or break the test, but `t` should be greater than `s` if there is a diff in definitions.  Maybe this helps to detect some bug. Some in other response mocks.

Reply:
Yeah, I do have proper sync tests with `s` and `t` are set correctly, in this test though, it is only for evaluation and matcher tests